libraries/GCS_MAVLink/GCS.h

```diff
@@ -354,6 +354,7 @@ class GCS_MAVLINK
     void send_rc_channels() const;
     void send_rc_channels_raw() const;
     void send_raw_imu();
+    void send_highres_imu();
```

Comment:
```suggestion
#if AP_MAVLINK_MSG_HIGHRES_IMU_ENABLED
    void send_highres_imu();
#endif
```

That define should be defined in `GCS_config.h` and disabled by default unless `BOARD_FLASH_SIZE > 1024`

This should go around every patch in here.